canary/producer-c/canary/CanaryStreamUtils.cpp

```diff
@@ -297,6 +297,21 @@ STATUS publishErrorRate(STREAM_HANDLE streamHandle, PCanaryStreamCallbacks pCana
     return retStatus;
 }
 
+STATUS pushStartUpLatency(PCanaryStreamCallbacks pCanaryStreamCallbacks)
```

Comment:
Just for my understanding.. We push the exact same value to two metrics? One that's aggregated and one that's not?

Reply:
Yes.